pkg/manager/meta/meta_manager_test.go

```diff
@@ -208,8 +209,8 @@ func TestMetaManagerSync(t *testing.T) {
 			pvcUpdateErr:  false,
 			pvUpdateErr:   false,
 			podChanged:    true,
-			pvcChanged:    true,
-			pvChanged:     true,
+			pvcChanged:    false,
```

Comment:
Why change this to false?

Reply:
PVC's labels are [inherited from StatefulSet label selector](https://github.com/kubernetes/kubernetes/blob/v1.10.0/pkg/controller/statefulset/stateful_set_utils.go#L152) other than from the pod, but the statefulset controller won't work in this unit test. So PVC and PV won't be changed.

Reply:
PVC and PV would be changed because some of the labels such as memberId、clusterId、storeId are synchronized from the pod.